tests/test-umockdev-record.vala

```diff
@@ -386,7 +386,6 @@ t_system_script_log_simple ()
     assert_cmpuint (logwords.length, Op.EQ, 3);
     assert_cmpstr (logwords[0], Op.EQ, "r");
     // should be quick, give it 5 ms at most
-    assert_cmpint (int.parse(logwords[1]), Op.LE, 5);
```

Comment:
It's fine to skip the tests on ARM with `#ifdef` and such, or multiply all times (script and timeout) by some factor. But I'd like to keep them at least for x86. The whole point of the test is to validate that the preload library actually implements the correct timing of the script.

Reply:
i think the problem is this strict time limits just aren't possible to achieve without controlling the testbed. the ubuntu autopkgtests, by nature of running in an openstack cloud, simply can't guarantee that *any* action will happen within a 5ms window.

I don't think #ifdef'ing for armhf would really be appropriate, since it's possible the problem is that armhf is slow on the ubuntu autopkgtest systems; but I don't know if that means it's slow on *all* armhf systems.

Probably the best approach then if you want to keep it is to have a minimum-time multipler as you mentioned, i'll see if i can adjust it.